graphics/libpixman-devel/Portfile

```diff
@@ -10,14 +10,13 @@ name                    libpixman-devel
 conflicts               libpixman
 set my_name             pixman
 version                 0.42.2
-revision                0
+revision                1
 checksums               rmd160  282e3f6fc956391df67398a414b083d221ffdf4d \
                         sha256  5747d2ec498ad0f1594878cc897ef5eb6c29e91c53b899f7f71b506785fc1376 \
                         size    652984
 
 categories              graphics
-platforms               darwin
-maintainers             {ryandesign @ryandesign}
+maintainers             {ryandesign @ryandesign} {mascguy @mascguy}
```

Comment:
In retrospect, this port may not change that often, so I don't necessarily need to be a co-maintainer.

I'm fine either way though.